debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -92,6 +92,16 @@
             .withValidation(Field::isBoolean)
             .withDescription("Whether invalid host names are allowed when using SSL. If true the connection will not prevent man-in-the-middle attacks");
 
+    public static final Field COMAPCT_DELETE_OPERATIONS = Field.create("kafka.force.compact.delete")
```

Comment:
s/COMAPCT/COMPACT/

Reply:
I'd prefer to avoid the "kafka" in there, to make clear it's not an option defined by Kafka itself. How about "tombstones.on.delete"?